### ISSUE TITLE
Check for nested ternary statements

### ISSUE THREAD

**Issue by jbrooksuk** [PHPCheckstyle/phpcheckstyle#36]
Being able to warn about nested ternary statements would be useful.

For example:

``` php
(1 === TRUE ? "Yay" : "Nay")
```

Would be fine, however:

``` php
(1 === TRUE ? ( 0 === FALSE ? "Foo" : "Bar" ) : "Nay")
```

Should report a warning that too many ternary statements are nested.
